app/controllers/products_controller.rb

```diff
@@ -12,5 +12,13 @@ def subscriptions
     @subscriptions=Stripe::Plan.list
   end 
 
-
+  def plan
+    plan = Stripe::Plan.create(
+      :name => "Basic Plan",
+      :id => "basic-monthly",
+      :interval => "month",
+      :currency => "usd",
+      :amount => 0,
+    )
+  end
```

Comment:
@heydanhey Whoa. I have no idea where that came in from. Mister Labatt might have had some influence on that one. I'll get rid of it tonight and re push. Sorry!